src/app/Services/backendResolver.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable, ErrorHandler, Component, OnInit } from '@angular/core';
+
+/*
+    Service for providing and receiving data from the REST interface that is used for general functions of the application.
+*/
+@Injectable({
+    providedIn: 'root'
+})
+export class BackendResolver {
+
+    backendPath: string;
+    readonly cookieKey = 'core-domain';
+
+    constructor() {
+        this.backendPath = localStorage.getItem(this.cookieKey);
+    }
+
+    setBackendPath(path: string) {
+        this.backendPath = path;
```

Comment:
should this also update the item in the localStorage?

Reply:
no, setBackendPath is called inside auth.service in login()